studio/src/pages/fact-checks/components/FactCheckForm.js

```diff
@@ -218,7 +218,13 @@ function FactCheckForm({ onCreate, data = {}, actions = {}, format }) {
         message="You have unsaved changes, are you sure you want to leave?"
       />
       {visible && (
-        <Modal visible={visible} onOk={handleOk} onCancel={handleCancel} maskClosable={false}>
+        <Modal 
+          visible={visible} 
+          onOk={handleOk} 
+          onCancel={handleCancel} 
```

Comment:
@vsumit89 Does `onOk` and `onCancel` handlers are required?

Reply:
okCancel is required